src/components/Book.js

```diff
@@ -0,0 +1,9 @@
+const Book = () => (
+  <li>
+    Book Two by Monica
+    {' '}
+    <button type="button">Delete Book</button>
+  </li>
```

Comment:
In order to have a dynamic render and to really make use of the power of React and React components, we need to use props that are passed from the parent component;
Look what this component should look like:
![image](https://user-images.githubusercontent.com/83596135/165976290-5e7dd6f1-41d8-4d7e-8e37-86865a1dcdc4.png)

Note: For the moment is not a problem if you deactivate linters regarding prop-types; we will learn about them later on;

- [ ] Kindly, refactor your Book component to make use of props from the parent component :+1:

Reply:
Thanks for the Review Nick. Again I thought since the form is not working, I shall add the props in the next step of the project, I did not know I needed to add the props for now. Let me do that.